server.js

```diff
@@ -9,12 +9,64 @@ app.use(cors());
 //Use this array as your (in-memory) data store.
 const bookings = require("./bookings.json");
 
-app.get("/", function (request, response) {
-  response.send("Hotel booking server.  Ask for /bookings, etc.");
+
+app.get("/",  (req, res) => {
+  res.send("Hotel booking server.  Ask for /bookings, etc.");
+});
+
+
+//Read all bookings
+app.get("/bookings", (req, res) => {
+  res.json(bookings);
 });
 
+//Create a new booking
+
+app.post("/bookings", (req, res)=>{
+
+    let newBooking = {
+
+      id: bookings.length + 1,
```

Comment:
This logic would not guarantee unique ids (consider what would happen if you delete and then add a new booking). Can you think of any ways that a unique id could be generated?